task/aws/resources/resource_bucket.go

```diff
@@ -29,10 +29,12 @@ type Bucket struct {
 
 func (b *Bucket) Create(ctx context.Context) error {
 	createInput := s3.CreateBucketInput{
-		Bucket: aws.String(b.Identifier),
-		CreateBucketConfiguration: &types.CreateBucketConfiguration{
-			LocationConstraint: types.BucketLocationConstraint(b.Client.Region),
-		},
+		Bucket:                    aws.String(b.Identifier),
+		CreateBucketConfiguration: &types.CreateBucketConfiguration{},
+	}
+
+	if b.Client.Region != "us-east-1" {
```

Comment:
is future-proofing required? `us-east-*`?

Reply:
See https://github.com/boto/boto3/issues/125